.github/workflows.src/build.inc.yml

```diff
@@ -153,11 +153,11 @@
         brew update
 
     - name: Install Rust toolchain
-      uses: dtolnay/rust-toolchain@d0e72ca3bfdc51937a4f81431ccbed269ef9f2a2
+      uses: dtolnay/rust-toolchain@master
```

Comment:
Please continue to use commit hashes.  Supply chain attaches via third-party actions are now commonplace.

Reply:
Switched to dtolnay/rust-toolchain@56f84321dbccf38fb67ce29ab63e4754056677e0